bubblewrap.c

```diff
@@ -330,13 +330,24 @@ monitor_child (int event_fd, pid_t child_pid)
         }
       else if (s == sizeof (struct signalfd_siginfo))
         {
+          pid_t pid;
+          int status;
+
           if (fdsi.ssi_signo != SIGCHLD)
             die ("Read unexpected signal\n");
 
           /* We may be getting sigchild from other children too. For instance if
              someone created a child process, and then exec:ed bubblewrap. Ignore them */
           if (fdsi.ssi_pid == child_pid)
             exit (fdsi.ssi_status);
```

Comment:
I'm not sure how that would work. If we read from the signalfd, then that reaps the child and we have to look at the result there. Alternatively we poll on the signalfd, but never read from it. Will the signalfd then poll non-readable if we reap the children with wait()?